wiki/Tournaments/oPQS/2/en.md

```diff
@@ -198,6 +198,53 @@ The osu! Philippines Quickfire Showdown 2023 is run by various community members
   1. [Eri Sasaki - Gate of Steiner (xbopost) [Milky-way Crossing]](https://osu.ppy.sh/beatmapsets/1010752#osu/2115799)
   2. [Eleharmonica remixed by kors k - Der Wald (kors k Remix) (Cheesecake) [mithew's Light Insane]](https://osu.ppy.sh/beatmapsets/581787#osu/1478254)
 
+## Match results
+
+### Day 1
```

Comment:
these headers should be in reverse order, see https://osu.ppy.sh/wiki/en/Tournaments/CIC/2019#match-results as an example